puppet/modules/slave/manifests/init.pp

```diff
@@ -251,6 +251,10 @@
     package { 'google-chrome-stable':
       ensure => latest,
     }
+
+    class { 'chromedriver':
+      ensure => latest,
```

Comment:
Looking at the source code this will install version 2.24 but the notes on https://chromedriver.storage.googleapis.com/2.33/notes.txt say we need 2.33 to work with the current chrome version.